osu.Game/Screens/Select/Carousel/CarouselBeatmapSet.cs

```diff
@@ -82,6 +82,9 @@ public override int CompareTo(FilterCriteria criteria, CarouselItem other)
 
                 case SortMode.Difficulty:
                     return compareUsingAggregateMax(otherSet, b => b.StarDifficulty);
+
+                case SortMode.Source:
+                    return string.Compare(BeatmapSet.Metadata.Source, otherSet.BeatmapSet.Metadata.Source, StringComparison.OrdinalIgnoreCase);
```

Comment:
Huge nit, but as order doesn't matter here, I think it would look organized to move this case below `SortMode.Author`, as a way of organizing string types above number types